cmd/connection_cleaner/main.go

```diff
@@ -33,18 +40,36 @@ func getRunningPods(dnsName string) (RunningPods, error) {
 	return runningPodMap, nil
 }
 
-func processConnection(dryRun bool, runningPods RunningPods, redisClient *redis.Client, account, nodeID, podName string) {
+func processConnection(dryRun bool, runningPods RunningPods, redisClient *redis.Client, account, nodeID, podName string, connCount *Metrics) {
 	if _, exists := runningPods[podName]; !exists {
 		fmt.Printf("Pod (%s) down!  This entry should be removed:  %s:%s\n", podName, account, nodeID)
 		if dryRun == false {
 			controller.UnregisterWithRedis(redisClient, account, nodeID, podName)
+			connCount.unregisterStaleConnectionFromRedis.Inc()
 		}
 	}
 }
 
+func pushValues(cfg *config.Config, connCount *Metrics) {
+	if err := push.New(cfg.PromPushGW, "receptor_controller_stale_job").
+		Collector(connCount.unregisterStaleConnectionFromRedis).
+		Push(); err != nil {
+		fmt.Println("Unable to push metrics to prometheus gateway")
+	}
+
+}
+
 func main() {
 	var podName = flag.String("pod-name", "receptor-gateway-internal", "DNS name of the internal gateway pod")
 	var dryRun = flag.Bool("dry-run", false, "Just report the stale connections.  No changes will be made.")
+
+	connCount := &Metrics{
+		unregisterStaleConnectionFromRedis: promauto.NewCounter(prometheus.CounterOpts{
+			Name: "receptor_controller_unregister_stale_connection_from_redis_count",
```

Comment:
Can we change the name to "receptor_controller_connection_cleaner_unregister_stale_connection_from_redis_count"?  I think I'd like to track this count separately from the "usual" method of removing stale connections.